libnavigation-ui/src/main/java/com/mapbox/navigation/ui/map/MapFpsDelegate.java

```diff
@@ -15,7 +15,7 @@
 
 class MapFpsDelegate implements OnTrackingModeChangedListener, OnTrackingModeTransitionListener {
```

Comment:
Why `MapFpsDelegateTest`s are commented out 🤔?

Reply:
Will be taken care separately in a different ticket.